lib/charms/prometheus_k8s/v0/prometheus_scrape.py

```diff
@@ -607,9 +607,6 @@ def render_alertmanager_static_configs(alertmanagers: List[str]):
                     "scheme": scheme,
```

Comment:
Should we document this?
```suggestion
                    # For https we still do not render a `tls_config` section because
                    # certs are expected to be made available by the charm via the
                    # `update-ca-certificates` mechanism.
                    "scheme": scheme,
```